KerbalStuff/thumbnail.py

```diff
@@ -1,12 +1,62 @@
 import os.path
-from typing import Tuple
 
 from PIL import Image
 
+from KerbalStuff.config import _cfg, _cfgi, site_logger
 
-def create(imagePath: str, thumbnailPath: str, thumbnailSize: Tuple[int, int]) -> None:
-    if not os.path.isfile(imagePath):
+
+def create(background_path: str, thumbnail_path: str) -> None:
+    if not os.path.isfile(background_path):
         return
-    im = Image.open(imagePath)
-    im.thumbnail(thumbnailSize, Image.ANTIALIAS)
-    im.save(thumbnailPath, 'jpeg', quality=50, optimize=True)
+
+    size_str = _cfg('thumbnail_size')
+    if not size_str:
+        size_str = "320x195"
+    size_str_tuple = size_str.split('x')
+    size = (int(size_str_tuple[0]), int(size_str_tuple[1]))
+
+    quality = _cfgi('thumbnail_quality')
+    if not quality or 0 > quality > 100:
```

Comment:
I don't think `0 > quality > 100` works as intended:

```
>>> x = 200
>>> (not x or 0 > x > 100)
False
```

Reply:
Oh interesting,so that logic is basically and-chained, not or-chained as I thought.
Thanks!